utilities/storage.py

```
@@ -1131,3 +1133,29 @@ def validate_file_exists_in_url(url):
         raise UrlNotFoundError(url_request=response)
 
     return True
+
+
+def persist_storage_class_default(default: bool, storage_class: StorageClass) -> None:
```

Ruff (0.14.13)

Boolean-typed positional argument in function definition

(FBT001)